assets/queries/terraform/azure/postgresql_server_threat_detection_policy_disabled/query.rego

```diff
@@ -0,0 +1,33 @@
+package Cx
+
+import data.generic.common as common_lib
+
+CxPolicy[result] {
+	pg := input.document[i].resource.azurerm_postgresql_server[name]
+
+	not common_lib.valid_key(pg, "threat_detection_policy")
+
+	result := {
+		"documentId": input.document[i].id,
+		"searchKey": sprintf("azurerm_postgresql_server[%s]", [name]),
+		"issueType": "MissingAttribute",
+		"keyExpectedValue": sprintf("'azurerm_postgresql_server[%s].threat_detection_policy' is defined and set to true", [name]),
```

Comment:
I think, since `threat_detection_policy`, is an object, doesn't makes sense to be true, I think the text would look better with the following changes:
```suggestion
		"keyExpectedValue": sprintf("'azurerm_postgresql_server[%s].threat_detection_policy' is a defined object", [name]),
```